<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateDownloadDialog</class>
 <widget class="QDialog" name="UpdateDownloadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>104</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>417</width>
    <height>104</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>417</width>
    <height>104</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Horus Updater</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblDownload">
     <property name="text">
      <string>Downloading update...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnRestart">
     <property name="text">
      <string>Restart Horus</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="pbDownload">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
